app/javascript/packs/stake_accounts/validator_row.js

```diff
@@ -100,6 +102,23 @@ var ValidatorRow = Vue.component('validatorRow', {
       });
       event.target.classList.add('active');
     },
+
+    // Set max_value position for root & vote distance charts
+    max_value_position(vector) {
+      var max_value = Math.max.apply(Math, vector);
+      var max_value_index = vector.indexOf(max_value).toFixed(2);
+      var vector_length = vector.length.toFixed(2);
+      var position = (max_value_index / vector_length * 100).toFixed(0);
+      position = Math.max.apply(Math, [position, 2])
+      // set max position for large numbers
+      if (max_value > 100000) {
+        position = Math.min.apply(Math, [position, 70])
+      } else if(max_value > 10000) {
+        position = Math.min.apply(Math, [position, 80])
+      }
+      return position + "%";
+    },
```

Comment:
I'm intentionally copying methods from rails helper to vue, so that we can drop rails views+helpers completely in the future. So yes, it's a copy for a moment, but with a plan to remove rails part (hopefully soon).